### PR TITLE
Fixed wishlist display when title is too long

### PR DESCRIPTION
**Pull request by nicosomb**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | If the wishlist title is too long, the display is broken
| Type?             | bug fix
| BC breaks?        | no
| Deprecations?     |  no
| Fixed ticket?     | Fixes https://github.com/PrestaShop/PrestaShop/issues/29213
| How to test?      | See https://github.com/PrestaShop/PrestaShop/issues/29213
| Possible impacts? | 

I also fixed the display in the popin when clicking on the "add to wishlist" button. 
Thank you @NeOMakinG 

Capture d’écran 2022-09-06 à 10 23 45
Capture d’écran 2022-09-06 à 10 24 02

### CODE REVIEW

**Review by NeOMakinG — APPROVED** [review]

**Review by kpodemski — APPROVED** [review]

**Review by matthieu-rolland — APPROVED** [review]

**Review by florine2623 — COMMENTED** [review]
Hello @nicosomb ,

Tested on 1.7.8.x, 8.0.x, develop.

I noticed an extra space when the name of wishlist is 1 big word:

Screenshot 2022-09-15 at 14 25 31

Other than that, checked "add to wishlist" popup. It is looking good.
Screenshot 2022-09-15 at 14 25 12


Could you check ?
Thanks!

**Comment by nicosomb** [comment]
Hello @florine2623. 
I tried on Firefox and Chrome and I can't reproduce it. Could you please send me more details about your environment?

**Comment by florine2623** [comment]
Hello @nicosomb ,

Sorry for the late answer.

I retested on Chrome 105.0.5195.125, Firefox and Opera and I reproduced the issue I'm having.
Screenshot 2022-10-06 at 11 38 38

More info about my shop:
Screenshot 2022-10-06 at 11 47 05


I have well installed the PR:
Screenshot 2022-10-06 at 11 44 46

I have deleted my browser cache and my PrestaShop cache as well.

Tested on 8.0.0. and 1.7.8.x, the issue is the same on both versions.

I hope the info I share can help ^^

**Comment by kpodemski** [comment]
ping @nicosomb :)

**Review by matks — CHANGES_REQUESTED** [review]
Changes have been requested

**Comment by nicosomb** [comment]
Hello @paulnoelcholot. 
I made new tries ... and I still can't reproduce the bug (firefox or chrome). 

Capture d’écran 2023-02-17 à 15 40 09

What can we do to go forward on this issue?

**Comment by leemyongpakvn** [comment]
@nicosomb I see **florine2623** and **paulnoelcholot** used narrower screen mobiles than yours. Maybe the bug happens with under 380px wide screen.

**Comment by nicosomb** [comment]
@paulnoelcholot @florine2623 I think that I fixed the bug.

**Review by MhiriFaten — COMMENTED** [review]
Hello @nicosomb ,

I have checked your PR, and here is my checklist : 

- checked with a sequence of characters **without** spaces :white_check_mark: 
- checked with a sequence of characters **with** spaces      :white_check_mark: 
- checked with Chrome and Firefox                                        :white_check_mark: 
- checked "add to wishlist" popup                                           :x:  


Fixed wishlist display when title is too long :white_check_mark: 

image

"add to wishlist" popup                                           :x:  


[Untitled_ Feb 21, 2023 4_27 PM.webm](https://user-images.githubusercontent.com/111756615/220387163-cda1020d-3ac0-4d6c-a379-6d91785b9d90.webm)

Could you recheck it please?

**Comment by nicosomb** [comment]
@MhiriFaten hmmm what is the issue on your screenshot?

**Comment by MhiriFaten** [comment]
@nicosomb , 
Contrary, the screenshot is for approve that the issue is well fixed in the wishlist (as it is marked just above the screenshot **Fixed wishlist display when title is too long** :white_check_mark: ).

The problem is in the popup wishlist as shown in the screenrecord.

**Comment by nicosomb** [comment]
ok 😄 I didn't see the video on my smartphone, but I can see it on my computer. I will have a look on this bug later.

**Comment by nicosomb** [comment]
@MhiriFaten I tried to reproduce with Firefox and Chrome, but I can't. The popin is well displayed on my side.

File: _dev/front/js/components/List/List.vue [hunk 1]
```diff
@@ -325,6 +326,8 @@
           letter-spacing: 0;
           line-height: 1.375rem;
           margin-bottom: 0;
+          max-width: 70%;
```
Comment (SharakPL) [inline _dev/front/js/components/List/List.vue:329]: Use of % is the reason for module not being consistent on different screen sizes so please don't use it. Best way to go is use flex on the parent, fixed width/flex-basis on buttons container and let the title fill the rest of the space.

Also it should be tested on v.3.0.0 released recently cause it already impacted wishlist modals.

**Pull request closed without merging**